apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1601,6 +1610,14 @@ export class RushConfiguration {
     return undefined;
   }
 
+  /**
+   * Finds the project that owns the specified POSIX relative path (e.g. apps/rush-lib).
```

Comment:
Some filesystems are case-insensitive (NTFS and both of the Mac filesystems, optionally).  

Is the caller expected to normalize the case of `posixRelativePath`?  If so, how?

Reply:
The expectation is that the caller obtained the file path from a tool like `git` that provided it with the correct casing. Otherwise the caller will have problems performing operations on Linux.
The lookup function does not care if the specified path actually exists, only if it maps to one of the defined project relative folders in a case-sensitive manner. It likewise does not care if the project folder itself actually exists.

I'll add to the documentation that it is case-sensitive.